app/styles/base.css

```diff
@@ -654,6 +654,79 @@ select:active {
 }
 
 /* ----------------------------------------
+ * Loading Animation
+ * ----------------------------------------
+ */
+
+#noVNC_loading_anim {
```

Comment:
Why not use our existing transition? The way I did it was add a new "state" class `noVNC_loading`, set that by default on `<html>` and have `ui.js` clear it on load.